### PR TITLE
Edit wanikani.proto and generate Protobuf

### PR DESCRIPTION
**Pull request by UInt2048**
Fix the issues in davidsansome/tsurukame#402 by re-generating from `wanikani.proto` and adding a setting.

### CODE REVIEW

File: ios/SettingsViewController.m [hunk 1]
```diff
@@ -205,6 +205,14 @@ - (void)rerender {
                                 on:Settings.allowSkippingReviews
                             target:self
                             action:@selector(allowSkippingReviewsSwitchChanged:)]];
+  [model addItem:[[TKMSwitchModelItem alloc]
+                     initWithStyle:UITableViewCellStyleDefault
+                             title:@"Minimize review penalty"
+                          subtitle:@"Treat reviews answered incorrect multiple times as if "
```
Comment (d108) [inline ios/SettingsViewController.m:211]: This subtitle does not seem to show in the UI.

**Review by d108 — CHANGES_REQUESTED** [review]
@UInt2048 有難う御座います for this!

I noted in the review the subtitle is not appearing for me.

Would you also mind submitting your changes in a single commit so that I can do a "rebase and merge," and then it should appear on top of mine in the PR to https://github.com/davidsansome/tsurukame?

File: ios/SettingsViewController.m (hunk 1 above)
Comment (UInt2048) [reply ios/SettingsViewController.m:211]: Fixed that - oops

**Comment by UInt2048** [comment]
@d108 I tried, but GitHub didn't cooperate - it said I had to re-merge. If you squash and merge, it should do what you want.

**Comment by d108** [comment]
@UInt2048 Your efforts are appreciated. I hope the result is to your satisfaction concerning the appearance of your commit.

**Pull request merged**